diff --git a/colors/pyte.vim b/colors/pyte.vim
index 7f173c4..1db0ff4 100644
--- a/colors/pyte.vim
+++ b/colors/pyte.vim
@@ -9,7 +9,7 @@ endif
 let colors_name = "pyte"
 
 if version >= 700
-  hi CursorLine guibg=#f6f6f6
+  hi CursorLine guibg=#ececec
   hi CursorColumn guibg=#eaeaea
   hi MatchParen guifg=white guibg=#80a090 gui=bold
 
@@ -19,7 +19,8 @@ if version >= 700
   hi TabLineSel guifg=black guibg=#f0f0f0 gui=italic,bold
 
   "P-Menu (auto-completion)
-  hi Pmenu guifg=white guibg=#808080
+  hi Pmenu     guifg=NONE guibg=NONE gui=NONE
+  hi PmenuSel  guifg=NONE guibg=#b0b0ff gui=NONE
   "PmenuSel
   "PmenuSbar
   "PmenuThumb
@@ -30,9 +31,10 @@ hi Title      guifg=#202020 gui=bold
 hi Underlined  guifg=#202020 gui=underline
 
 
-hi Cursor    guifg=black   guibg=#b0b4b8
-hi lCursor   guifg=black   guibg=white
-hi LineNr    guifg=#ffffff guibg=#c0d0e0
+hi Cursor           guifg=black   guibg=#b0b4b8
+hi lCursor          guifg=black   guibg=white
+hi LineNr           guifg=#ffffff guibg=#b0c0d0
+hi CursorLineNr     guifg=#ffffff guibg=#b1c1d1
 
 hi Normal    guifg=#404850   guibg=#f0f0f0
 
@@ -41,24 +43,23 @@ hi StatusLineNC guifg=#506070 guibg=#a0b0c0 gui=italic
 hi VertSplit guifg=#a0b0c0 guibg=#a0b0c0 gui=NONE
 
 " hi Folded    guifg=#708090 guibg=#c0d0e0
-hi Folded    guifg=#a0a0a0 guibg=#e8e8e8 gui=italic
+hi Folded    guifg=#808080 guibg=#e4e4e4 gui=italic
 
 hi NonText   guifg=#c0c0c0 guibg=#e0e0e0
 " Kommentare
-hi Comment   guifg=#a0b0c0               gui=italic
+hi Comment   guifg=#708090               gui=italic
 
 " Konstanten
 hi Constant  guifg=#a07040
 hi String    guifg=#4070a0 
 hi Number    guifg=#40a070
 hi Float     guifg=#70a040
-"hi Statement guifg=#0070e0 gui=NONE
 " Python: def and so on, html: tag-names
-hi Statement  guifg=#007020 gui=bold
+hi Statement  guifg=#007020 gui=none
 
 
 " HTML: arguments
-hi Type       guifg=#e5a00d gui=italic
+hi Type       guifg=#38888e gui=italic
 " Python: Standard exceptions, True&False
 hi Structure  guifg=#007020 gui=italic
 hi Function   guifg=#06287e gui=italic
@@ -66,18 +67,18 @@ hi Function   guifg=#06287e gui=italic
 hi Identifier guifg=#5b3674 gui=italic
 
 hi Repeat      guifg=#7fbf58 gui=bold
-hi Conditional guifg=#4c8f2f gui=bold
+hi Conditional guifg=#4c8f2f gui=none
 
 " Cheetah: #-Symbol, function-names
 hi PreProc    guifg=#1060a0 gui=NONE
 " Cheetah: def, for and so on, Python: Decorators
-hi Define      guifg=#1060a0 gui=bold
+hi Define      guifg=#1060a0 gui=none
 
 hi Error      guifg=red guibg=white gui=bold,underline
 hi Todo       guifg=#a0b0c0 guibg=NONE gui=italic,bold,underline
 
 " Python: %(...)s - constructs, encoding
-hi Special    guifg=#70a0d0 gui=italic
+hi Special    guifg=#70a0d0 gui=none
 
 hi Operator   guifg=#408010
 
@@ -91,4 +92,14 @@ hi DiffText guifg=NONE guibg=#f0c8c8 gui=italic,bold
 hi DiffAdd guifg=NONE guibg=#c0e0d0 gui=italic,bold
 hi DiffDelete guifg=NONE guibg=#f0e0b0 gui=italic,bold
 
+if exists('+colorcolumn')
+    hi ColorColumn guibg=#eaeaea ctermbg=254
+endif
+
+" Gitgutter
+hi SignColumn            guibg=#b0c0d0
+hi GitGutterAdd          guibg=#b0c0d0 guifg=#ffffff gui=bold
+hi GitGutterChange       guibg=#b0c0d0 guifg=#ffffff gui=bold
+hi GitGutterDelete       guibg=#b0c0d0 guifg=#ffffff gui=bold
+hi GitGutterChangeDelete guibg=#b0c0d0 guifg=#ffffff gui=bold
 
